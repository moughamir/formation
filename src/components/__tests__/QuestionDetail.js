describe('QuestionDetail Component', () => {

  it('Should not regress', () => {

  });
});
