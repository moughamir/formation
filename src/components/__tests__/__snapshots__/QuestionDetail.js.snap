// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionDetail Component The Display element Should not regress 1`] = `[Function]`;
